scss/style.scss

```
@@ -0,0 +1,369 @@
+@import "__fonts.scss";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+body {
+  width: 100%;
+}
```

можно не прописывать, `body `и так на всю ширину
```suggestion
body {
}
```